CHANGES.md

```diff
@@ -1,5 +1,11 @@
 ## [HEAD]
 
+#### New features
```

Comment:
Do we really need to further sub-divide this doc? I was hoping to keep each release small enough that we wouldn't have to.